lib/navigation.php

```diff
@@ -280,7 +284,7 @@ function gutenberg_migrate_menu_to_navigation_post( $new_name, $new_theme, $old_
 	update_option( 'fse_navigation_areas', $area_mapping );
 }
 
-add_action( 'switch_theme', 'gutenberg_migrate_menu_to_navigation_post', 200, 3 );
+add_action( 'switch_theme', 'gutenberg_migrate_menu_to_navigation_post', 99, 3 );
```

Comment:
Because of this change: https://github.com/WordPress/wordpress-develop/pull/1865/files#diff-4112a9a0bb18713bc2fc7868ceffe4403d765722f605e42b313ef2cdc218177fR685

I learned it's more typical to use priority at most 99 in core and that's how it was backported.